kolibri/core/analytics/tasks.py

```diff
@@ -25,6 +26,12 @@
 def _ping(started, server, checkrate):
     try:
         ping_once(started, server=server)
+        try:
+            ping_error_reports.enqueue(args=(server,))
```

Comment:
This will only happen if the task has a static id - enqueuing a new instance of the task will not give this error otherwise.

Given that we're not worrying about a static id, we can remove the try except here.